tests/modules/bowtie2/align/test.yml

```diff
@@ -1,83 +1,21 @@
-- name: bowtie2 align single-end
-  command: nextflow run ./tests/modules/bowtie2/align -entry test_bowtie2_align_single_end -c ./tests/config/nextflow.config -c ./tests/modules/bowtie2/align/nextflow.config
+- name: bowtie2 align test_bowtie2_align_single_end
+  command: nextflow run tests/modules/bowtie2/align -entry test_bowtie2_align_single_end -c tests/config/nextflow.config
```

Comment:
```suggestion
  command: nextflow run ./tests/modules/bowtie2/align -entry test_bowtie2_align_single_end -c ./tests/config/nextflow.config -c ./tests/modules/bowtie2/align/nextflow.config
```

Reply:
We need to keep that all in as they are because this is what we use in the modules template in nf-core/tools [here](https://github.com/nf-core/tools/blob/e0d00d3e09a996f59eb5ce6ebc4795452a5b1d39/nf_core/module-template/tests/test.yml#L4)

Reply:
This file was auto generated by `nf-core` tools.  I suppose this is something that also should be fixed there then?

Reply:
Appears like we need to update [this line](https://github.com/nf-core/tools/blob/e0d00d3e09a996f59eb5ce6ebc4795452a5b1d39/nf_core/modules/test_yml_builder.py#L168) to reflect what we have in the module. Not sure whether this happens when you have single/multiple tests to run.

Would you mind creating an issue please? We need to fix this before the next release.

Reply:
A minimal reproducible example would be great too :)

Reply:
https://github.com/nf-core/tools/issues/1562